changelogs/fragments/3164-zypper-support-transactional-updates.yaml

```
@@ -0,0 +1,2 @@
+minor_changes:
+  - zypper - with transactional updates in use, e.g. on MicroOS, zypper can't be simply used to install packages. Parts of the file-system are read-only. Prefix zypper-commands with `/sbin/transactional-update --continue --drop-if-no-change --quiet run` if transactional-updates are detected (https://github.com/ansible-collections/community.general/issues/3159).
```

```suggestion
  - zypper - prefix zypper commands with ``/sbin/transactional-update --continue --drop-if-no-change --quiet run`` if transactional updates are detected (https://github.com/ansible-collections/community.general/issues/3159).
```
I would shorten this a bit.

---

applied and squashed the commit

---

BTW, there's no need to squash commits. We squash on merge. Squashing during the review process mainly makes it harder to trace through the changes.